src/Arcus.Messaging.Pumps.ServiceBus/Resiliency/CircuitBreakerServiceBusMessageHandler.cs

```diff
@@ -61,11 +61,9 @@ await CircuitBreaker.PauseMessageProcessingAsync(messageContext.JobId, opt =>
                     opt.MessageIntervalDuringRecovery = options.MessageIntervalDuringRecovery;
                     opt.MessageRecoveryPeriod = options.MessageRecoveryPeriod;
                 });
-                await AbandonMessageAsync();
```

Comment:
Taken this out as the test showed that sometimes messages get abandonned twice in the same receive operation - one time here, and one time by the message router.